src/types/Person.ts

```diff
@@ -9,3 +9,22 @@ export interface Person {
   mother?: Person,
   father?: Person,
 }
+
+export enum ErrorOption {
+  NoError,
+  Wrong,
+  Empty,
+}
+
+export enum Sex {
+  All = 'all',
+  MALE = 'm',
+  FEMALE = 'f',
+}
+
+export enum SortBy {
+  ByName = 'name',
+  BySex = 'sex',
+  ByBorn = 'born',
+  ByDied = 'died',
```

Comment:
```suggestion
  Name = 'name',
  Sex = 'sex',
  Born = 'born',
  Died = 'died',
```

Reply:
Sex is already declarated. I wanted to do like you. But i also have enum Sex